connector_search_engine/tests/test_all.py

```diff
@@ -170,6 +170,17 @@ def test_unlink_record(self):
         self.partner.unlink()
         self.assertEqual(self.partner_binding.state, "to_delete")
 
+    def test_archive_record(self):
+        self.partner.active = False
+        self.assertEqual(self.partner_binding.state, "to_delete")
```

Comment:
I don't think this test is necessary as it's the exact same two lines as the start of `test_archive_unarchive_record()`